applications/rag/outputs.tf

```diff
@@ -42,3 +42,8 @@ output "frontend_ip_address" {
   description = "Frontend gloabl IP address"
   value       = module.frontend.frontend_ip_address
 }
+
+output "ray_dashboard_uri" {
```

Comment:
same check output of already existing ray-cluster

Reply:
This is separate only for the dashboard, it can not be used to submit job